Sources/QuickTests/FunctionalTests/PendingTests.swift

```diff
@@ -5,24 +5,35 @@ import Nimble
 import QuickTestHelpers
 #endif
 
-var oneExampleBeforeEachExecutedCount = 0
-var onlyPendingExamplesBeforeEachExecutedCount = 0
+var isBeforeSuiteCalled = false
+var isAfterSuiteCalled = false
+var isBeforeEachCalled = false
+var isAfterEachCalled = false
 
 class FunctionalTests_PendingSpec: QuickSpec {
     override func spec() {
-        xit("an example that will not run") {
-            expect(true).to(beFalsy())
+        describe("a describe block containing only pending examples") {
+            beforeSuite { isBeforeSuiteCalled = true }
+            beforeEach { isBeforeEachCalled = true }
+            pending("an example that will not run") {}
+            afterEach { isAfterEachCalled = true }
+            afterSuite { isAfterSuiteCalled = true }
```

Comment:
So, this is *almost* what we want. However, the `beforeSuite` and `afterSuite` tests here will pass even without your changes. This is because this test suite contains non-pending examples. To really test these, you'd need an entirely new test target in the Quick workspace -- just like the QuickFunctionalTests target.

I think we should test this in a new QuickPendingTests target.